.circleci/config.yml

```diff
@@ -1273,6 +1280,7 @@ workflows:
       - node-cassandra
       - node-connect
       - node-couchbase
+      - node-cucumber
```

Comment:
This is missing from the `requires` section below so that it's included in the code coverage report.

Reply:
done in https://github.com/DataDog/dd-trace-js/pull/1251/commits/e549e1619b61d69c2ec50ced3b94c430a80cadc6